CHANGES.rst

```diff
@@ -331,6 +331,9 @@ astropy.io.fits
 
 - Fix the ``fitscheck`` script for updating invalid checksums, or removing
   checksums. [#6571]
+  - Strip leading/trailing white-space from latex lines to avoid issues when matching ``\begin{tabular}`` statements.  This is done by introducing a new ``LatexInputter`` class to override the ``BaseInputter``.
```

Comment:
somehow the rebase got messed up on the changelog, could you move it up under `astropy.io.ascii` and add the PR number to it in a new commit?